N3.js/src/main/scala/org/w3/banana/n3js/model.scala

```diff
@@ -0,0 +1,7 @@
+package org.w3.banana
+package n3js
+package model
```

Comment:
The problem is related to eclipse that has a melt down if directories and packages do not match - I can see why you did what you did, it's quite an instinctive approach.

If you remove model, then in N3js.scala you get ```type BNode = BNode```

maybe create model/Model.scala ?